### PR TITLE
Pending changes for Beta 11 of React on Rails

### PR DESCRIPTION
**Pull request by justin808**

### CODE REVIEW

File: lib/webpacker/railtie.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Webpacker::Engine < ::Rails::Engine
   end
 
   initializer "webpacker.bootstrap" do
-    if defined?(Rails::Server) || defined?(Rails::Console)
+    if (defined?(Rails::Server) || defined?(Rails::Console)) && Webpacker.config.config_path.exist?
```
Comment (gauravtiwari) [inline lib/webpacker/railtie.rb]: Why we need to check config file? The idea is to fail when running server or console if none exist. For other things like running react on rails installer this block won't run.

File: lib/webpacker/commands.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@ def clobber
   end
 
   def bootstrap
+    return unless config.config_path.exist?
```
Comment (dhh) [inline lib/webpacker/commands.rb]: I don't think we want #bootstrap to be a silent no-op if the config is missing.

Comment (justin808) [inline lib/webpacker/commands.rb]: @dhh @gauravtiwari How about if I submit another PR with the check only here, and print a  warning message?

At this level of the code, is a "puts" to stdout OK?

Comment (dhh) [inline lib/webpacker/commands.rb]: I don't think we should have any check inside the method of #bootstrap. I'd rather have it where #bootstrap is called. Also don't want to do any puts anywhere inside these classes. They should all use the logger. Only the rake tasks can put.
